library/src/main/camera2/com/google/android/cameraview/Camera2.java

```diff
@@ -385,7 +407,6 @@ void setFacing(int facing) {
         mFacing = facing;
         if (isCameraOpened()) {
             stop();
-            start();
```

Comment:
It seems like it may be needed as a way to reset the camera, maybe I deleted by mistake. I'll add it back